src/js/components/stock-movement-wizard/outbound/AddItemsPage.jsx

```diff
@@ -854,7 +829,14 @@ class AddItemsPage extends Component {
     const lineItems = _.filter(formValues.lineItems, item => !_.isEmpty(item));
 
     if (_.some(lineItems, item => !item.quantityRequested || item.quantityRequested === '0')) {
-      this.confirmSave(() => this.saveItems(lineItems));
+      this.confirmSave(() => {
+        this.saveItems(lineItems);
+        const filteredLineItems = lineItems.filter(item => item.quantityRequested > 0);
+        this.setState({
+          values: { lineItems: filteredLineItems },
+          totalCount: filteredLineItems.length,
```

Comment:
and here respecitvely:
```suggestion
          totalCount,
```